selenium_test.py

```diff
@@ -0,0 +1,273 @@
+from csv import DictWriter
+from collections import defaultdict
+
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+import pandas as pd
+import time
+import os
+
+
+KEYWORDS_FILE = 'textfile.txt'
+LOGIN_FILE = 'login.txt'
+RANKED_OUTPUT_FILE = 'Output.csv'
+START_YEAR = "1917"
+END_YEAR = "1971"
+
+
+class Articles:
+    def __init__(self, title, description, category, keyword, link ):
+        self.title = title
+        self.description = description
+        self.category = category
+        self.keyword = keyword
+        self.link = link
+
+    def to_dict(self):
+        return {
+            'Title': self.title,
+            'Description': self.description,
+            'Category': self.category,
+            'Keyword': self.keyword,
+            'Link': self.link,
+
+        }
+
+
+
+
+
+def read_login_info():
+    with open(LOGIN_FILE) as text_file:
+        loginInfo = text_file.read().split(',')
+        return loginInfo
+
+
+def read_keywords():
+    with open(KEYWORDS_FILE) as fd:
+        return [keyword.strip() for keyword in fd.readlines()]
+
+
+def scrape_economist(keyword):
+    allResults = []
+
+    path = os.getcwd() + "/geckodriver"
+    driver = webdriver.Firefox(executable_path=path)
```

Comment:
I think the better approach here is to assume that the user knows how to set up `geckodriver` in their path, and just do `driver = webdriver.Firefox()`.